src/metrics/captures.ts

```diff
@@ -148,6 +198,20 @@ export class KDRatioMetric implements Metric {
         this.KDAssistsMap[move.uas].kills++;
         this.KDAssistsMap[move.capture.uas].deaths++;
 
+        this.KDAssistsValuesMap[move.capture.uas].deaths++;
+
+        // identify kill values based on captured piece type
+        if (move.capture.type === 'p' || move.capture.type === 'k') {
+          this.KDAssistsValuesMap[move.uas].valueKills++;
+        } else if (move.capture.type === 'n' || move.capture.type === 'b') {
+          this.KDAssistsValuesMap[move.uas].valueKills += 3;
+        } else if (move.capture.type === 'r') {
+          this.KDAssistsValuesMap[move.uas].valueKills += 5;
+        } else if (move.capture.type === 'q') {
+          this.KDAssistsValuesMap[move.uas].valueKills += 9;
```

Comment:
instead of doing this why don't we just track kills by ambiguous piece type? Then we can calculate this with an aggregate function

Reply:
the data could be interesting on which piece kills which - i thought we may have wanted to track that

Reply:
wouldn't it be easier to track each uas kills and then aggregate to determine ambig piece type kills?